connector/datadogconnector/go.mod

```diff
@@ -15,17 +15,18 @@ require (
 )
 
 require (
-	github.com/DataDog/datadog-agent/pkg/obfuscate v0.49.0-rc.2 // indirect
-	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.49.0-rc.2 // indirect
-	github.com/DataDog/datadog-agent/pkg/trace v0.50.0-devel.0.20231030140030-aa8a206e6ac7 // indirect
-	github.com/DataDog/datadog-agent/pkg/util/cgroups v0.49.0-rc.2 // indirect
-	github.com/DataDog/datadog-agent/pkg/util/log v0.49.0-rc.2 // indirect
-	github.com/DataDog/datadog-agent/pkg/util/pointer v0.49.0-rc.2 // indirect
-	github.com/DataDog/datadog-agent/pkg/util/scrubber v0.49.0-rc.2 // indirect
+	github.com/DataDog/datadog-agent/pkg/obfuscate v0.50.0-rc.7 // indirect
+	github.com/DataDog/datadog-agent/pkg/remoteconfig/state v0.50.0-rc.7 // indirect
+	github.com/DataDog/datadog-agent/pkg/trace v0.50.0-rc.7 // indirect
```

Comment:
For some reason these deps are left out by renov bot. They should be updated together.